js_modules/dagster-ui/packages/ui-core/src/app/AppTopNav.tsx

```diff
@@ -118,6 +124,11 @@ export const AppTopNav: React.FC<Props> = ({
     ];
   };
 
+  const {reloading, tryReload} = useRepositoryLocationReload({
```

Comment:
(Read through this hook to make sure it's safe to have multiple copies of it on a page and it seems fine)